From e14fa0a774804769b5c8b9b67a7b040b1a5a8bbc Mon Sep 17 00:00:00 2001
From: Peter O'Connor <peter@solus-project.com>
Date: Mon, 16 Oct 2017 11:53:18 +1100
Subject: [PATCH] Solus tweaks to the default config

 - Remove toolbox icon from desktop
 - Decrease desktop icon sizes
 - Add shortcuts for Software Center in menu

Signed-off-by: Peter O'Connor <peter@solus-project.com>
---
 applets/kicker/package/contents/config/main.xml       | 2 +-
 applets/kickoff/package/contents/config/main.xml      | 4 ++--
 containments/desktop/package/contents/config/main.xml | 4 ++--
 3 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/applets/kicker/package/contents/config/main.xml b/applets/kicker/package/contents/config/main.xml
index c1884e8..b72e154 100644
--- a/applets/kicker/package/contents/config/main.xml
+++ b/applets/kicker/package/contents/config/main.xml
@@ -38,7 +38,7 @@
     </entry>
     <entry name="favoriteApps" type="StringList">
       <label>List of general favorites. Supported values are menu id's (usually .desktop file names), special URLs that expand into default applications (e.g. preferred://browser), document URLs and KPeople contact URIs.</label>
-      <default>preferred://browser,kontact.desktop,systemsettings.desktop,org.kde.dolphin.desktop,ktp-contactlist.desktop,org.kde.kate.desktop,org.kde.discover</default>
+      <default>preferred://browser,kontact.desktop,systemsettings.desktop,org.kde.dolphin.desktop,ktp-contactlist.desktop,org.kde.kate.desktop,solus-sc.desktop</default>
     </entry>
     <entry name="favoriteSystemActions" type="StringList">
       <label>List of system action favorites.</label>
diff --git a/applets/kickoff/package/contents/config/main.xml b/applets/kickoff/package/contents/config/main.xml
index be031dc..6d8303b 100644
--- a/applets/kickoff/package/contents/config/main.xml
+++ b/applets/kickoff/package/contents/config/main.xml
@@ -20,7 +20,7 @@
     </entry>
     <entry name="favorites" type="StringList">
       <label>List of general favorites. Supported values are menu id's (usually .desktop file names), special URLs that expand into default applications (e.g. preferred://browser), document URLs and KPeople contact URIs.</label>
-      <default>preferred://browser,kontact.desktop,systemsettings.desktop,org.kde.dolphin.desktop,ktp-contactlist.desktop,org.kde.kate.desktop,org.kde.discover.desktop</default>
+      <default>preferred://browser,kontact.desktop,systemsettings.desktop,org.kde.dolphin.desktop,ktp-contactlist.desktop,org.kde.kate.desktop,solus-sc.desktop</default>
     </entry>
     <entry name="favoritesPortedToKAstats" type="Bool">
       <label>Are the favorites ported to use KActivitiesStats to allow per-activity favorites</label>
@@ -28,7 +28,7 @@
     </entry>
     <entry name="systemApplications" type="StringList">
       <label>List of applications at the top of the "Computer" tab.</label>
-      <default>systemsettings.desktop,org.kde.kinfocenter.desktop,org.kde.discover.desktop</default>
+      <default>systemsettings.desktop,org.kde.kinfocenter.desktop,solus-sc.desktop</default>
     </entry>
     <entry name="useExtraRunners" type="Bool">
       <label>Whether to use additional KRunner plugins to produce results in the search.</label>
diff --git a/containments/desktop/package/contents/config/main.xml b/containments/desktop/package/contents/config/main.xml
index 6830fb0..d172180 100644
--- a/containments/desktop/package/contents/config/main.xml
+++ b/containments/desktop/package/contents/config/main.xml
@@ -100,7 +100,7 @@
     </entry>
     <entry name="iconSize" type="Int">
       <label>The icon size to use for Folder View icons. One of the list of Plasma icon sizes.</label>
-      <default>4</default>
+      <default>3</default>
     </entry>
     <entry name="textLines" type="Int">
       <label>The (maximum) number of lines of text to use below Folder View icons.</label>
@@ -134,7 +134,7 @@
     </entry>
     <entry name="showToolbox" type="Bool">
       <label>Whether to show the Desktop Toolbox. Only used when serving as containment.</label>
-      <default>true</default>
+      <default>false</default>
     </entry>
     <entry name="pressToMove" type="Bool">
       <label>Whether to activate widget handles and widget move mode by long press. Only used when serving as containment.</label>
-- 
2.16.2

